src/index.js

```diff
@@ -47,12 +47,16 @@ let ElasticsearchProvider = (config = { request: {} }) => {
           }
 
       let client = config.getClient()
+      let child = client.child({
+        headers: requestOptions.headers,
+        requestTimeout: requestOptions.requestTimeout,
```

Comment:
does this mean that we shouldn't be passing `requestOptions` to `search` anymore?

Reply:
Well this encompasses all the needs that we are using for requestOptions, but you could always pass other things to search within those options that we just aren't using.

Reply:
got it - so should we take headers/requestTimeout off before passing it along?

Reply:
given that we were passing it along before, it's probably fine. Merging :)